src/domains/gps-app/api_product/payments-service/_base_policy.xml

```diff
@@ -0,0 +1,18 @@
+<policies>
+    <inbound>
+        <base />
+        <ip-filter action="forbid">
+            <!-- appgateway-snet  -->
+            <address-range from="10.1.128.0" to="10.1.128.255" />
```

Comment:
this check should be true only for "internal" api , combined to appgw deny paths

Reply:
what do you means when you say "appgw deny paths"? It must be added a new file/resource (if yes, what kind of file/resource?) or an existent file must be updated (if yes, what file?) ?

Reply:
`10.1.128.0` ... `10.1.128.255` it's appgw subnet . This check says `if some request coming from this ips, stop it`
And this is a further control at [deny appgw path](https://github.com/pagopa/pagopa-infra/blob/b0633a471b9bbfe0f93b5c1731d821245db46fb3/src/core/env/prod/terraform.tfvars#L93) see [here](https://learn.microsoft.com/en-us/azure/application-gateway/rewrite-url-portal) for doc